api/platform/types.go

```diff
@@ -384,6 +384,8 @@ type ClusterFeature struct {
 	// +optional
 	EnableCilium bool
 	// +optional
+	EnableContainerRuntime string
```

Comment:
EnableContainerRuntime to ContainerRuntime
should not be string, should define a ContainerRuntimeType